pvlib/spectrum/mismatch.py

```diff
@@ -77,8 +81,18 @@ def get_example_spectral_response(wavelength=None):
     return sr
 
 
+@deprecated(
+    since="0.11",
+    removal="0.12",
+    name="pvlib.spectrum.get_am15g",
+    alternative="pvlib.spectrum.get_standard_spectrum",
+    addendum=(
+        "The new function reads more data. Use it with "
+        + "standard='ASTM G173-03' and extract the 'global' column."
```

Comment:
Let's follow our own advice and change `calc_spectral_mismatch_field` to use the new function so that it doesn't emit deprecation warnings :)

Reply:
Should we fix the spectra used? Or let it use the default one?